helm-charts/sep-service/templates/deployment.yaml

```diff
@@ -30,10 +30,8 @@ spec:
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repo }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
-          args:
-            - --sep-server
-            - --stellar-observer
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args: [{{ join ", " .Values.deployment.args | default ( "--sep-server" | quote ) }}]
```

Comment:
💡  Can we include the `--stellar-observer` as a default value as well?

I'll review/change this when I separate the observer into a different pod.

Reply:
Sure, I've added it to the default